tests/models/test_layers_utils.py

```diff
@@ -120,6 +120,18 @@ def test_upsample_default(self):
         expected_slice = torch.tensor([-0.2173, -1.2079, -1.2079, 0.2952, 1.1254, 1.1254, 0.2952, 1.1254, 1.1254])
         assert torch.allclose(output_slice.flatten(), expected_slice, atol=1e-3)
 
+    def test_upsample_bfloat16(self):
```

Comment:
Let's have a decorator like 
https://github.com/huggingface/diffusers/blob/14a1b86fc7de53ff1dbf803f616cbb16ad530e45/src/diffusers/utils/testing_utils.py#L362

And mark this test with that? I think that's safer.

Reply:
yup added one